NitroxPatcher/Patches/Dynamic/StoryHandTarget_OnHandClick_Patch.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Reflection;
+using NitroxClient.Communication.Abstract;
+using NitroxClient.GameLogic;
+using NitroxClient.GameLogic.Simulation;
+using NitroxClient.MonoBehaviours.Gui.HUD;
+using NitroxModel.Core;
+using NitroxModel.DataStructures;
+using NitroxModel.Helper;
+using NitroxModel.Packets;
+
+namespace NitroxPatcher.Patches.Dynamic
```

Comment:
you can use file scoped namespace, that allows to remove the extra brackets


namespace NitroxPatcher.Patches.Dynamic;